### PR TITLE
81 face variations

### PR DESCRIPTION
**Pull request by psugihara**
Added new face expressions. I think I like 'mania' and 'hitcycle' most. The 'time' mode feels too random. The nice thing about 'mania' is it is very noticeable even when there's a lot of chaos because it's most chaotic.

Downsized to (200x200 via `sips -Z 200 *.png`) and minified pngs via tinypng.

Here are the 3 modes you can try in the anybody options:
```
  // faceRotation: 'time' | 'hitcycle' | 'mania'
  // time: start sleepy and get happier as time goes on
  // hit: rotate to a new face each time (expression is starLvl % 3)
  // mania: when body is hit, cycle wildly until end of game
```

addresses #81

### CODE REVIEW

File: src/visuals.js [hunk 1]
```diff
@@ -413,7 +451,7 @@ export const Visuals = {
     }
 
     // make the timer bigger as time runs out
-    if (secondsLeft < 10 && this.scoreSize < 420) {
+    if (secondsLeft <= 9 && this.scoreSize < 420) {
```
Comment (psugihara) [inline src/visuals.js:454]: this resize animation a little cleaner if it starts at a 1 digit number

File: package.json [hunk 2]
```diff
@@ -92,7 +92,6 @@
     "hardhat-gas-reporter": "*",
     "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
-    "p5": "^1.8.0",
```
Comment (psugihara) [inline package.json:95]: i think we're not using this 🧹

**Comment by okwme** [comment]
made an update that fixes the timer when using live data, need to track starting time point to calculate correctly.

also updated the hitcycle logic a bit. Now it doesn't cycle through all three but just the current and next face once it's hit. When it's at level 3 it toggles last and first.

made small change to withering so that only maxxed out stars wither, otherwise you may not actually see the final face for some bodies because if they're the last to be hit they'll disappear as soon as they reach final face. Reminded me that we discussed changing withering logic as well so that they keep moving until the final proof is generated then they freeze. Maybe good to make them bodies until that point too. I'm gonna start the #88 PR now, will need to be integrated into withering eventually, may need to rethink logic even further.

**Review by okwme — APPROVED** [review]

**Comment by psugihara** [comment]
cool, lgtm

**Pull request merged**